Oxide.Core/RemoteConsole/RemoteConsole.cs

```diff
@@ -1,11 +1,13 @@
+using Newtonsoft.Json;
```

Comment:
Using statements are in an odd order still. System would generally be at the top to match current formatting usage, the rest A-Z.